src/utils/index.ts

```diff
@@ -570,76 +476,50 @@ export const getTokenLogoURL = (inputAddress: string, chainId?: ChainId): string
 
   let imageURL
 
+  imageURL = store
```

Comment:
wow, default image was moved to this. nice job bro

Reply:
Yes, we find in tokenlist first, if not, switch case will be the fallback logic. We can actually remove switch case logic in the future.